lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -530,12 +530,27 @@ func processAnonymousVolumes(h *string, volumes map[string]struct{}, client *cli
 
 func processSpecifiedVolumes(volumes []string) ([]volumeFields, error) {
```

Comment:
Hey, I know you didn't name this, but can you add a function comment explaining a 'specified' volume?  The terminology difference with what docker actually uses may confuse people.